Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/dialogs/changeDocStrucType.xhtml

```diff
@@ -28,28 +27,38 @@
-        <h:form id="dialogEditDocStrucTypeForm">
+        <h:form id="dialogEditDocStrucTypeForm"
+                style="margin-top: -32px;">
 
-            <!-- Select the new type -->
-            <p:selectOneMenu id="docStructEditTypeSelection"
-                             class="input"
-                             value="#{DataEditorForm.changeDocStrucTypeDialog.docStructType}"
-                             required="true"
-                             immediate="true"
-                             style="width: 180px; margin-left: 33px;">
-                <f:selectItems value="#{DataEditorForm.changeDocStrucTypeDialog.docStructTypes}"/>
-            </p:selectOneMenu>
+            <h:panelGroup layout="block"
+                          styleClass="select-selector">
 
-            <!-- Press OK when finish -->
-            <p:commandButton id="addDocStruc"
-                             value="#{msgs.apply}"
-                             action="#{DataEditorForm.changeDocStrucTypeDialog.editDocStruc}"
-                             styleClass="primary right"
-                             icon="fa fa-check fa-lg"
-                             iconPos="right"
-                             style="margin-right: 33px; margin-top: 29px;"
-                             update="structureTreeForm:logicalTree,paginationForm:paginationWrapperPanel,metadataAccordion:logicalMetadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel"
-                             oncomplete="updateMetadataEditorView(false);"/>
+                <!-- Select the new type -->
+                <p:selectOneMenu id="docStructEditTypeSelection"
+                                 class="input"
+                                 value="#{DataEditorForm.changeDocStrucTypeDialog.docStructType}"
+                                 required="true"
+                                 immediate="true">
+                    <f:selectItems value="#{DataEditorForm.changeDocStrucTypeDialog.docStructTypes}"/>
+                </p:selectOneMenu>
+
+                <!-- Press OK when finish -->
+                <h:panelGroup layout="block">
+                    <p:commandButton id="changeDocStruc"
+                                     value="#{msgs.apply}"
+                                     action="#{DataEditorForm.changeDocStrucTypeDialog.editDocStruc}"
+                                     styleClass="primary right"
+                                     icon="fa fa-check fa-lg"
+                                     iconPos="right"
+                                     update="structureTreeForm:logicalTree,paginationForm:paginationWrapperPanel,metadataAccordion:logicalMetadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel"
+                                     oncomplete="updateMetadataEditorView(false);"/>
+                    <p:commandButton value="#{msgs.cancel}"
+                                     icon="fa fa-times fa-lg"
+                                     iconPos="secondary"
```

Comment:
Should this be `right`?

Reply:
You are right, it should of course be `right`, not `secondary`. 

Funny enough, it seems any value but `left` results in the icon on the right side:
- `iconPos="left"` -> Icon on the **left**
- `iconPos="right"` -> Icon on the **right**
- `iconPos=""` -> Icon on the **right**
- `iconPos="foobar"` -> Icon on the **right**
- no `iconPos` attribute at all -> Icon on the **left**